docs/introduction.rst

```diff
@@ -1,73 +1,57 @@
+Click to download Nion Swift or for :ref:`installation` instructions.
 
-Key Features
+Click here to get started fast with our :ref:`basic-use`.
 
-- Data handling of 1D plots, 2D images, 1D and 2D collections of plots and images, and sequences.
-- Live computations that can be chained during acquisition or live parameter adjustment.
-- Open source, cross platform (macOS, Windows, Linux), and Python based scientific data processing.
-- Low latency, high throughput data acquisition platform.
+User Interface
+==============
+The user interface is split up into a central section of the window called the workspace and the surrounding area for utility panels. 
 
-Data Items and Display Items
-----------------------------
-|AppName| operates on data arrays, which include 2D images and 1D spectra, which are generally called *Data Items*.
+The central workspace is for display panels which can either display a single piece of data (also called a data item) or can display multiple pieces of data side by side in one of the browser formats. For more information, see :ref:`display-panels`.
 
-Data items represent data arrays with anywhere from one to five dimensions. The data array associated with a data item can be 1D or 2D data and then organized into 1D or 2D collections or 1D sequences.
+The utility panels surrounding the central workspace allow you to make adjustments to a selected data item in a display panel. For more information on their specific functions, see :ref:`user-interface`.
 
-A spectrum image would be a 2D collection of 1D data. This would be typical data acquired from a STEM microscope spectrum imaging acquisition.
+Data and Display
+================
+A piece of data stored in the program is called a data item. Data items can be dragged into display panels to be displayed. Once displayed, they become display items. Display items can be modified and viewed to see various aspects or values of the original data item. Read more in :ref:`data-items`.
 
-Data Items are displayed using one or more *Display Items*.
+If a data item includes complex components, it will need to be reduced into a scalar image or line plot in order to be displayed. For more info, see :ref:`data-items`.
 
-See :ref:`data-items`.
+Sometimes, data items can be part of a larger 'stack' of data which will require higher level organization that the data panel can provide. See :ref:`data-items` for more info.
 
-Projects
---------
-Data items are organized into projects. Projects are stored on disk and take the form of an index file and an associated data directory.
+The INFO PANEL shows you the location and value of the data in a display panel under your cursor. You can choose how you would like the :ref:`data-items` (coordinate system) to be defined in the inspector panel.
```

Comment:
Did you forget a link to the Info Panel here? (INFO PANEL)

Overall, this section seems to be mixing technical details (i.e. converting complex to scalar, displaying values in the info panel, etc.) with an overview (describing what the overall purpose is).